src/binding-signaler.js

```diff
@@ -1,16 +1,11 @@
-import {sourceContext} from 'aurelia-binding';
+import {sourceContext, signalBindings} from 'aurelia-binding';
+import * as LogManager from 'aurelia-logging';
 
 export class BindingSignaler {
-  signals = {};
 
   signal(name: string): void {
-    let bindings = this.signals[name];
-    if (!bindings) {
-      return;
-    }
-    let i = bindings.length;
-    while (i--) {
-      bindings[i].call(sourceContext);
-    }
+    LogManager.getLogger('binding-signaler')
+      .warn(`Binding signaler signal has been deprecated. Please use "import {signalBindings} from 'aurelia-binding' instead.`);
+    signalBindings(name);
   }
```

Comment:
What do you think about adding a constructor to BindingSignaler and putting the log message there?

Reply:
Sure, was just waiting for feedback so it's a bit ... lazy there 😆